document/core/exec/runtime.rst

```diff
@@ -158,6 +170,8 @@ and collects runtime representations of all entities that are imported, defined,
      \MITABLES & \tableaddr^\ast, \\
      \MIMEMS & \memaddr^\ast, \\
      \MIGLOBALS & \globaladdr^\ast, \\
+     \MIELEMS & \elemaddr^\ast, \\
```

Comment:
As mentioned above, I think these need to be optional addresses to represent indices of active segments correctly.